cl/search/types.py

```diff
@@ -85,3 +88,16 @@
 SaveDocumentResponseType = tuple[str, ESDictDocument]
 
 SearchAlertHitType = tuple[Alert, str, list[ESDictDocument], int]
+
+
+class EventTable(Enum):
+    DOCKET = "docket"
+    DOCKET_ENTRY = "de"
+    RECAP_DOC = "rd"
```

Comment:
I try to avoid abbreviations like this so that we're always verbose and the codebase can be counted on to be verbose. Therefore:

```suggestion
    RECAP_DOCUMENT = "rd"
```

Reply:
Great, Yeah I've changed the event types and now they look as:
```
DOCKET = "search.Docket"
DOCKET_ENTRY = "search.DocketEntry"
RECAP_DOCUMENT = "search.RECAPDocument"
```